pkg/types/spec/validations.go

```diff
@@ -473,7 +473,8 @@ func computeValidation() *cr.StructFieldValidation {
 
 func autoscalingValidation(kind userconfig.Kind) *cr.StructFieldValidation {
```

Comment:
This func can be converted to a var, and the `minReplicas` variable can be replaced by `0`